README.md

```diff
@@ -10,16 +10,11 @@ The goal of the UI is to provide the users a convenient way of interacting with
 In the `.env` file at root project directory, you have to define the following env variables:
```

Comment:
That is not correct. As soon as .env is being committed then this should refer ONLY to `REACT_APP_RIALTO_SUBSTRATE_PROVIDER` and `REACT_APP_RIALTO_SUBSTRATE_PROVIDER` and only for .env.local (or else people will be pushing the .env to PRs)

Reply:
i'm not getting exactly what you mean, but at least you made me realize that variables are incorrectly called.

They should be:
REACT_APP_SUBSTRATE_PROVIDER_CHAIN_1=wss://wss.rialto.brucke.link
REACT_APP_SUBSTRATE_PROVIDER_CHAIN_2=wss://wss.millau.brucke.link

Reply:
Anyway, please take into account that the other purpose of the .env file is for deployment purposes. So in this case updating the .env is right because a new variable was added and some of them were removed. Also we will split specific `.env` files for deploying specific bridge pair.